docs/source/complex_numbers.rst

```diff
@@ -4,7 +4,7 @@ Complex Numbers
 ===============
 
 Complex numbers are numbers that can be expressed in the form :math:`a + bj`, where a and b are real numbers,
-and *j* is a solution of the equation :math:`x^2 = -1`. Complex numbers frequently occur in mathematics and
+and *j* called the imaginary unit,satisfies the equation :math:`j^2 = -1`. Complex numbers frequently occur in mathematics and
```

Comment:
missing space between "unit," and "satisfies"?

Reply:
So do you suggest to leave the proper space and then make a new pull request ? :D

Reply:
It should fixed before landing. Please update this PR.

Reply:
do I have to follow those steps I found in stackoverflow ? 
(I'm noobie and I dont want to make any mistake)

git reset --hard <commit key of the pull request>
Did my changes in code I wanted to do
git add
git commit --amend
git push -f origin <name of the remote branch of pull request>

Reply:
I think I done it, now I'm waiting again . 
As you can imagine I'm new here and trying to understand how git works and github works